B2B. YVES. Wrong info in printing return details (#1246)
spryker/spryker-docs: docs/scos/user/features/202204.0/product-barcode-feature-overview.md

A barcode is a square or rectangular image consisting of a series of parallel black lines (bars) and white spaces of varying widths that can be read by a scanner and printed. Barcodes are applied to entities as a means of quick identification.
![Barcode example](https://spryker.s3.eu-central-1.amazonaws.com/docs/Features/Product+Management/Barcode+Generator/Barcode+Generator+Feature+Overview/barcode.png)

By default, barcodes are generated based on product SKUs using the [Code128](https://en.wikipedia.org/wiki/Code_128) format. 

> Though, Spryker provides highly customizable solutions through plugins with the help of which the setup can be changed.

Reviewer: ```suggestion
Though, Spryker highly provides customizable solutions through plugins that let you change the setup.
```

Barcodes are dynamically generated for concrete products, which ensures that barcodes are immediately valid.

Also, depending on whether a customer chooses a product bundle or configurable bundle, the number of barcodes differs:
* If a customer purchases a product bundle, it's always assigned only one barcode, regardless of the quantity of the products within the bundle.
* If a customer purchases a configurable bundle, each product within the configurable bundle is assigned a separate unique barcode.

The following image demonstrates how barcodes are assigned depending on whether it's a product bundle or a configurable bundle.

![product-bundle-vs-configurable-bundle](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/user/features/product-barcode-feature-overview/product-bundle-versus-configurable-bundle.png)

> You can see the barcodes in the Back Office, in the **Catalog&nbsp;<span aria-label="and then">></span> Product Barcodes** section. The section is designed as a review; thus, no actual actions are performed here. The barcode is generated automatically once a new concrete product is added.

Reviewer: ```suggestion
You can view the barcodes in the Back Office, in the **Catalog&nbsp;<span aria-label="and then">></span> Product Barcodes** section. You can not perform any other actions on the barcodes, as they are generated automatically once a new concrete product is added to the system.
```

The Barcode Generator can be used for any kind of entity, and by default, Spryker provides a solution for products.

> **What is a barcode?**

Reviewer: delete. Also, please delete the "What is the barcode" heading in the very beginning of this document, after the first sentence. The new "What is the barcode" section does not make sense
